network-api/networkapi/wagtailpages/pagemodels/base.py

```diff
@@ -742,6 +776,15 @@ class Meta(TranslatableMixin.Meta, WagtailOrderable.Meta):
 
 
 class Homepage(FoundationMetadataPageMixin, Page):
+
+    donate_banner = models.ForeignKey(
+        DonateBanner,
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="homepage_donate_banner",
+        help_text="Select the Donate Banner to display on the homepage",
```

Comment:
**TODO**: If we continue with this implementation, we would want to add help text, etc., letting the user know that the DonateBanner set would be sitewide.